src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int MAX_TASK = 100;
```

Comment:
Perhaps this could be plural i.e. `MAX_TASKS`?
```suggestion
    private static final int MAX_TASKS = 100;
```